packages/eslint-plugin-pf-codemods/src/rules/v6/colorPropsReplacedColors/colorProps-replaced-colors.test.ts

```diff
@@ -0,0 +1,112 @@
+const ruleTester = require("../../ruletester");
+import * as rule from "./colorProps-replaced-colors";
+import {
+  ValidTests,
+  InvalidTests,
+  createInvalidTest,
+  createValidTest,
+} from "../../helpers/testHelpers";
+
+const components = ["Banner", "Label"];
+const validTests: ValidTests = [];
+const invalidTests: InvalidTests = [];
+components.forEach((component) => {
+  validTests.push(createValidTest(`<${component} color="gold" />`));
+  validTests.push(createValidTest(`<${component} color="cyan" />`));
+  validTests.push(
+    createValidTest(
+      `import { ${component} } from '@patternfly/react-core'; <${component} color="red" />`
+    )
+  );
+
+  const type = "JSXOpeningElement";
+  invalidTests.push(
+    createInvalidTest(
+      `import { ${component} } from '@patternfly/react-core'; <${component} color="gold" />`,
+      `import { ${component} } from '@patternfly/react-core'; <${component} color="yellow" />`,
+      [
+        {
+          message: `The \`color\` prop on ${component} has been updated to replace "gold" with "yellow".`,
+          type,
+        },
+      ]
+    )
+  );
+  invalidTests.push(
+    createInvalidTest(
+      `import { ${component} } from '@patternfly/react-core'; <${component} color="cyan" />`,
+      `import { ${component} } from '@patternfly/react-core'; <${component} color="teal" />`,
+      [
+        {
+          message: `The \`color\` prop on ${component} has been updated to replace "cyan" with "teal".`,
+          type,
+        },
+      ]
+    )
+  );
+  invalidTests.push(
+    createInvalidTest(
+      `import { ${component} } from '@patternfly/react-core'; const color = 'gold'; <${component} color={color} />`,
+      `import { ${component} } from '@patternfly/react-core'; const color = 'gold'; <${component} color="yellow" />`,
```

Comment:
I think we would ideally update the assignment in situations like this rather than just the use, not a blocker though.